src/Standard/TIP-3/core-description.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Core description
+sidebar_position: 0
+---
+
+# Distributed Token
+
+## Abstract
+
+The following standard describes the basic idea about distributed fungible token architecture.
+
+## Motivation
+
+The suggested standard differs considerably from Ethereum ERC20 and other smart contract token standards with single registry due to its distributed nature related to Everscale blockchain particularities. Given that Everscale has a storage fee, using an existing ERC20 standard design would cause excessive maintenance costs. Also, ERC20 is somewhat incompatible with the sharding architecture. Therefore, a Distributed Token standard is preferable.
+
+The ERC20 sharding implementation (with an idea to simply shard its registry) has drawbacks mainly related to complicated and expansive management. TIP-3 is fully distributed and implies separate storage of each user’s balance.
+
+## Architecture
+
+General information about token is stored in the [token root](#token-root) contract. Each token holder has its own instance of [token wallet](#token-wallet) contract. Token transfers SHOULD be implemented in P2P fashion, between sender and receiver token wallets.
+
+### Token root
+
+Token root contract stores the general information about the token, e.g. name, symbol, decimals and so on. Any contract can ask root token to deploy an instance of token wallet contract.
```

Comment:
`Any contract can ask root token to deploy an instance of token wallet contract.` 
No details about it later or in sub proposals. No requirements for the root.